### PR TITLE
To-do Project added

### PR DESCRIPTION
**Pull request by Raj142**
Create a To-do list project using Javascript and added it to list of projects.

### CODE REVIEW

**Comment by 991rajat** [comment]
@Raj142 Fill the description Properly and Link the issue please

File: script.js [hunk 1]
```diff
@@ -54,6 +54,12 @@ let projectData = [
     projectImage: "assets/GIFs/bouncingBalls.gif",
     projectUrl: "public/BouncingBalls.html",
   },
+  {
+    projectName: "To-do list",
+    projectImage: "assets/GIFs/todo.gif",
+    projectUrl: "public/Todo.html",
```
Comment (991rajat) [inline script.js:60]: Fix this as well, project is not loading properly.
Make it todo.html

**Review by 991rajat — CHANGES_REQUESTED** [review]
Fix the submit button

File: script.js (hunk 1 above)
Comment (Raj142) [reply script.js:60]: sure

**Pull request closed without merging**